### PR TITLE
Add source / website of the listing at the end of description while d…

### PR DESCRIPTION
**Pull request by AkSu-HEIDI**
…isplaying.  Make the link clickable.

### CODE REVIEW

File: src/Path/SubPages/Listing.js [hunk 1]
```diff
@@ -516,6 +520,14 @@ const Listing = () => {
                   {t("description")}
                 </h1>
                 <Description content={description} />
+                {sourceId === 3 && (
```
Comment (AjUm-HEIDI) [inline src/Path/SubPages/Listing.js:523]: Add the comparision to a comstant file. Add a constant called source

File: src/Components/ListingsCard.jsx [hunk 2]
```diff
@@ -6,110 +6,127 @@ import { useNavigate } from "react-router-dom";
...
+    <div
+      onClick={(e) => {
+        e.stopPropagation();
+        if (listing.sourceId === 1 || listing.showExternal === 0) {
+          navigateTo(
+            `/Listing?listingId=${listing.id}&cityId=${listing.cityId}${
+              terminalView ? "&terminalView=true" : ""
+            }`
+          );
+        } else if (listing.sourceId === 3 || listing.showExternal === 1) {
```
Comment (AjUm-HEIDI) [inline src/Components/ListingsCard.jsx:27]: Move the comparision of the source id to a constant file

Comment (AjUm-HEIDI) [reply src/Components/ListingsCard.jsx:27]: And here the logic should be &&

Comment (AjUm-HEIDI) [reply src/Components/ListingsCard.jsx:27]: You can simplify this if else block into 1 if else

if ((source == 3 && showExternal) || source == 2) navigate to external link
else navigate to internal link

**Review by AjUm-HEIDI — CHANGES_REQUESTED** [review]

File: src/Components/ListingsCard.jsx (hunk 2 above)
Comment (AkSu-HEIDI) [reply src/Components/ListingsCard.jsx]: Done

**Review by AjUm-HEIDI — APPROVED** [review]

**Pull request merged**